ddtrace/ext/memcached.py

```diff
@@ -1,4 +1,4 @@
 CMD = "memcached.command"
 SERVICE = "memcached"
-TYPE = "memcached"
+TYPE = "cache"
```

Comment:
Interesting. This used to be `AppTypes.cache` but was changed in the same PR that added memcached support. https://github.com/DataDog/dd-trace-py/pull/511#discussion_r207932657